lusidtools/commands/commands.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import sys
+import lusidtools.lpt as lpt
+import os
+import ast
+
+
+def find_tools():
+    root = lpt.__path__[0]
+    for filename in os.listdir(root):
+        if filename.endswith('.py'):
```

Comment:
To remove the nesting here we can do something like valid_files = [filename for filename in os.listdir(root) if filename.endswith('.py')]

Then

for file in valid files:
   etc.